src/helm/benchmark/runner.py

```diff
@@ -106,6 +117,12 @@ def run_one(self, run_spec: RunSpec):
         run_path: str = os.path.join(self.runs_path, run_spec.name)
         ensure_directory_exists(run_path)
 
+        if os.path.exists(os.path.join(run_path, "scenario_state.json")):
```

Comment:
I don't see where `skip_completed_runs` is being used. Should we check it here?

Reply:
Thanks for catching the bug. Added the flag check.